admin/app/views/workarea/admin/segment_rules/fields/_last_order.html.haml

```diff
@@ -1,3 +1,3 @@
-%span Ordered within
+%span= t('workarea.admin.segment_rules.fields.last_order.ordered_in')
```

Comment:
I had left these untranslated for similar reason to the madlibs style in discounts (writing it in another language may not translate into the UI in the same way). So let's allow the variant to handle localizing these partials.